js/ui/endSessionDialog.js

```diff
@@ -799,7 +799,7 @@ var EndSessionDialog = new Lang.Class({
         let updatePrepared = this._pkOfflineProxy ? this._pkOfflineProxy.UpdatePrepared : false;
         let updatesAllowed = this._updatesPermission && this._updatesPermission.allowed;
 
-        _setCheckBoxLabel(this._checkBox, dialogContent.checkBoxText);
+        _setCheckBoxLabel(this._checkBox, dialogContent.checkBoxText || '');
```

Comment:
Nitpick: I believe this would be better if all entries of `DialogContent` had the `checkBoxText` property set to `null` or empty string.

Reply:
In general I prefer that style too. I wrote the patch this way because there are quite a few properties like this one (only defined in a few of the entries, with existence checks wherever they are used) and I didn't really want to get into changing that.

This patch was [merged upstream](https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/245) in this form, so let's take it as-is.